app/controllers/evaluations_controller.rb

```diff
@@ -1,5 +1,7 @@
 class EvaluationsController < ApplicationController
   before_action :require_resort_or_group_leader
+  before_action :require_application_or_evaluation_season
```

Comment:
Some tests are failing at this callback because, they don't set the season to application season.

Reply:
I looked it again, we set the season to application season in rails_helper.rb, but the SystemAttribute factory is not correct. If you set the value [here](https://github.com/kir-dev/pek-next/blob/77dadd6f41e2149445d9ff9190aa6f0899b21e02/test/factories/system_attributes.rb#L15) to `SystemAttribute::APPLICATION_SEASON` it will work as expected.